spec.bs

```diff
@@ -467,17 +467,17 @@ dictionary AuctionAdConfig {
   USVString trustedScoringSignalsURL;
   sequence<USVString> interestGroupBuyers;
   Promise<any> auctionSignals;
-  any sellerSignals;
-  USVString directFromSellerSignals;
+  Promise<any> sellerSignals;
+  Promise<USVString> directFromSellerSignals;
   unsigned long long sellerTimeout;
   unsigned short sellerExperimentGroupId;
   USVString sellerCurrency;
   Promise<record<USVString, any>> perBuyerSignals;
-  record<USVString, unsigned long long> perBuyerTimeouts;
+  Promise<record<USVString, unsigned long long>> perBuyerTimeouts;
   record<USVString, unsigned short> perBuyerGroupLimits;
   record<USVString, unsigned short> perBuyerExperimentGroupIds;
   record<USVString, record<USVString, double>> perBuyerPrioritySignals;
-  record<USVString, USVString> perBuyerCurrencies;
+  Promise<record<USVString, USVString>> perBuyerCurrencies;
   sequence<AuctionAdConfig> componentAuctions = [];

```

Comment:
Why this particular set of fields? Are the rest coming in another PR, or should there be a Note about which ones have to be synchronous?

Reply:
This should match what the implementation has[1] (ref: https://chromium-review.googlesource.com/c/chromium/src/+/4619514/4/third_party/blink/renderer/modules/ad_auction/auction_ad_config.idl)

As to why does it have the particular things? Well, initial set was based on industry feedback, then other things were also made promises when they seemed similar enough. 

[1] Except that perBuyerCumulativeTimeouts is missing from the spec.